format/Message.fbs

```diff
@@ -42,6 +42,12 @@ struct FieldNode {
   null_count: long;
 }
 
+enum CompressionType:byte {
+  UNCOMPRESSED,
+  LZ4,
```

Comment:
Lets please not repeat https://issues.apache.org/jira/browse/PARQUET-1241, I'm not sure i have all the context on it but can you please be specific on LZ4 format?

Reply:
Yes, I will document